Brownie pm is attempting to load `.env` from installed package

### Environment information

* `brownie` Version: 1.18.1
* Python Version: 3.8.12
* OS: osx

### What was wrong?
My tests now fail after adding a `.env` file to my project and specifying that file in `brownie-config.yml`.
They fail when attempting to setup a fixture that utilises OpenZeppelin's ERC20 contract.

Here is the fixture:
```
@pytest.fixture(scope="session")
def oz(pm):
    project = pm("OpenZeppelin/openzeppelin-contracts@4.5.0/")
    return project
```
Brownie config:
```
dependencies:
  - OpenZeppelin/openzeppelin-contracts@4.5.0
  - OpenZeppelin/openzeppelin-contracts-upgradeable@4.5.0
dotenv: .env
compiler:
  solc:
    version: 0.8.9
    optimizer:
      enabled: true
      runs: 200
    remappings:
      - "@openzeppelin=OpenZeppelin/openzeppelin-contracts@4.5.0"
      - "@openzeppelin-upgradeable=OpenZeppelin/openzeppelin-contracts-upgradeable@4.5.0"`
```
And the error:
```
    @pytest.fixture(scope="session")
    def oz(pm):
>       project = pm("OpenZeppelin/openzeppelin-contracts@4.5.0/")
E       ValueError: Dotenv specified in config but not found at path: ~/.brownie/packages/OpenZeppelin/openzeppelin-contracts@4.5.0/.env
```

Why is brownie trying to load an `.env` file from the package? any ideas on how to fix this, or is it a bug?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.18.1](https://github.com/eth-brownie/brownie/tree/v1.18.1) - 2022-02-15
 ### Fixed
 - Correctly modify chain time when using `chain.mine` with Ganache v7 ([#1438](https://github.com/eth-brownie/brownie/pull/1438))
+-  Ensure `.env` file exists for installed packages ([#1504](https://github.com/eth-brownie/brownie/pull/1504))
 
 ## [1.18.0](https://github.com/eth-brownie/brownie/tree/v1.18.0) - 2022-02-14
 ### Added

--- a/brownie/project/main.py
+++ b/brownie/project/main.py
@@ -811,6 +811,7 @@ def _install_from_ethpm(uri: str) -> str:
     try:
         new(str(install_path), ignore_existing=True)
         ethpm.install_package(install_path, uri)
+        Path.touch(install_path / ".env")
         project = load(install_path)
         project.close()
     except Exception as e:
@@ -887,6 +888,8 @@ def _install_from_github(package_id: str) -> str:
 
             with install_path.joinpath("brownie-config.yaml").open("w") as fp:
                 yaml.dump(brownie_config, fp)
+
+            Path.touch(install_path / ".env")
 
         project = load(install_path)
         project.close()

